README.md

```diff
@@ -1,54 +1,283 @@
+
+The best free C# IDE is [VS Community 2017](https://visualstudio.microsoft.com/vs/community), which you'll need to work on the project efficiently. Open `BizHawk.sln` with VS to start.
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+### GNU+Linux and macOS
+
+*Compiling* requires MSBuild and *running* requires Mono and WINE, but **BizHawk does not run under WINE** — only the bundled libraries are required.
+
+If you use GNU+Linux, there might be a `bizhawk-git` package or similar in the same repo as the main package. If it's available, installing it will automate the build process.
+
+Building is as easy as:
+```sh
+git clone https://github.com/TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+# or ssh: git clone git@github.com:TASVideos/BizHawk.git BizHawk_master && cd BizHawk_master
+msbuild /p:Configuration=Release BizHawk.sln
+```
+
+Remove the `/p:...` flag from MSBuild if you want debugging symbols.
+
+If your distro isn't listed under *Installing* above, `libblip_buf` probably isn't in your package repos. You can easily [build it yourself](https://gitlab.com/TASVideos/libblip_buf/blob/unified/README.md).
+
+Once built, see the *Installing* section, substituting the repo's `output` folder for the download.
+
+Again, if your distro isn't listed there, you might get an "Unknown distro" warning in the terminal, and BizHawk may not open or may show the missing dependencies dialog. You may need to add your distro to the case statement in the script, setting `libpath` to the location of `d3dx9_43.dll.so` (please do share if you get it working).
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+## Usage
+
+TODO
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+### TASing
+
+This section refers to BizHawk specifically. For resources on TASing in general, see [Welcome to TASVideos](http://tasvideos.org/WelcomeToTASVideos.html).
+
+TODO
+
+[Commandline](http://tasvideos.org/Bizhawk/CommandLine.html)
+
+[CompactDiscInfoDump](http://tasvideos.org/Bizhawk/CompactDiscInfoDump.html)
+
+[Rerecording](http://tasvideos.org/Bizhawk/Rerecording.html)
+
+[TAS movie file format](http://tasvideos.org/Bizhawk/TASFormat.html)
+
+[to top](https://github.com/TASVideos/BizHawk/blob/master/README.md#bizhawk)
+
+### Cores
+
+A *core* is what we call the smaller bits of software that emulate just one system or family of systems, e.g. NES/Famicom. For the most part, there's a "best" core for each system, based on accuracy, but there's currently a bit of overlap in the cores BizHawk uses as noted below.
+
+System | Core | Alt. Core | Special Core
+--:|:--|:--|:--
+Amstrad CPC | CPCHawk† ||
+Apple II | Virtu ||
+Atari 2600 | Atari2600Hawk ||
+Atari 7800 | A7800Hawk ||
+Atari Lynx | Handy ||
+Commodore 64 | C64Hawk ||
+ColecoVision | ColecoHawk ||
+Game Boy / Color | GBHawk | Gambatte | SameBoy (on SNES with SGB)
```

Comment:
SameBoy is for Super GameBoy, not for GB/GBC. Deserves a separate line.

Reply:
But it runs Game Boy / Color games? Gonna need a third opinion on that one

Reply:
GB, GBC, SGB, and GBA are modes you can run GB/GBC games in. IIRC we haven't enabled anything but SGB mode for SameBoy. @nattthebear halp

Reply:
If we decide SGB is its own system, Dual Neopop and the whole column are going too (I can't imagine it's popular).

Reply:
I think SGB should be separated. bsnes emulates that too.

Reply:
ok then